src/tokens/token.ts

```diff
@@ -158,6 +160,7 @@ export const tokenDataByNetwork: Record<
     dFRAX: "0x8a1112afef7f4fc7c066a77aabbc01b3fff31d47",
 
     GEAR: "0xBa3335588D9403515223F109EdC4eB7269a9Ab5D",
+    OHM: "0x383518188c0c6d7730d91b2c03a03c837814a899",
```

Comment:
Best to keep it with other normal tokens, some tests may stop after reaching dTokens